Dart/src/com/jetbrains/lang/dart/ide/runner/server/webdev/DartWebdevParameters.java

```diff
@@ -77,8 +78,14 @@ public void check(@NotNull final Project project) throws RuntimeConfigurationErr
       throw new RuntimeConfigurationError(DartBundle.message("old.dart.sdk.for.webdev", MIN_DART_SDK_VERSION_FOR_WEBDEV, sdkVersion));
     }
 
-    if (StringUtil.isEmptyOrSpaces(myHtmlFilePath)) {
-      throw new RuntimeConfigurationError(DartBundle.message("path.to.html.file.not.set"));
+    // check html file
+    final VirtualFile htmlVirtualFile = getHtmlFile();
+
+    // check computed working directory
+    final String workingDirVirtualFile = computeProcessWorkingDirectory(project);
+    final String expectedPubspecParentPath = htmlVirtualFile.getParent().getParent().getPath();
```

Comment:
`dart_root/web/subdir/something.html` works fine, I'll update the check accordingly.